src/entities/course/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const ALIAS_QUARTER_REGEXP = new RegExp(/-\d{4}q\d/, 'i');
+export const ALIAS_QUARTER_REGEXP = new RegExp(/-\d{4}(?:-?[Qq][1-4])?$/, 'i');
```

Comment:
Please add tests for the recent courses to make sure we're not missing anything
https://app.rs.school/api/v2/courses